components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher/src/main/java/org/wso2/carbon/apimgt/rest/api/publisher/impl/WorkflowsApiServiceImpl.java

```diff
@@ -0,0 +1,101 @@
+package org.wso2.carbon.apimgt.rest.api.publisher.impl;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.wso2.carbon.apimgt.core.api.APIPublisher;
+import org.wso2.carbon.apimgt.core.api.WorkflowExecutor;
+import org.wso2.carbon.apimgt.core.api.WorkflowResponse;
+import org.wso2.carbon.apimgt.core.exception.APIManagementException;
+import org.wso2.carbon.apimgt.core.exception.APIMgtResourceNotFoundException;
+import org.wso2.carbon.apimgt.core.exception.ExceptionCodes;
+import org.wso2.carbon.apimgt.core.models.Workflow;
+import org.wso2.carbon.apimgt.core.models.WorkflowStatus;
+import org.wso2.carbon.apimgt.core.util.APIMgtConstants;
+import org.wso2.carbon.apimgt.core.workflow.WorkflowExecutorFactory;
+import org.wso2.carbon.apimgt.rest.api.common.dto.ErrorDTO;
+import org.wso2.carbon.apimgt.rest.api.common.util.RestApiUtil;
+import org.wso2.carbon.apimgt.rest.api.publisher.*;
+import org.wso2.carbon.apimgt.rest.api.publisher.dto.*;
+import org.wso2.carbon.apimgt.rest.api.publisher.utils.MappingUtil;
+import org.wso2.carbon.apimgt.rest.api.publisher.utils.RestAPIPublisherUtil;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.wso2.carbon.apimgt.rest.api.publisher.NotFoundException;
+
+import javax.ws.rs.core.Response;
+
+public class WorkflowsApiServiceImpl extends WorkflowsApiService {
+    private static final Logger log = LoggerFactory.getLogger(WorkflowsApiServiceImpl.class);
+
+    @Override
+    public Response workflowsUpdateWorkflowStatusPost(String workflowReferenceId, WorkflowDTO body)
```

Comment:
Shall we add the method comment here? We cannot avoid this although the code is generated.